app/views/helpers/fieldConstructor.scala.html

```diff
@@ -21,7 +21,7 @@
 '_help -> "Custom help",                        (shows help text)
 '_labelClass -> "form-label-bold",              (extra class for labels)
 '_legend -> true,                               (shows legend/label)
-'_optional -> true,                             (shows optional/required for screen readers)
+'_optional -> true,                             (shows optional/required for screen readers - deprecated)
```

Comment:
According to Dan - we should no longer make the screen reader reads `hidden` text